README.md

```diff
@@ -68,9 +68,16 @@ webdriver-manager update      // Downloads the latest binaries.
 webdriver-manager start       // Starts the selenium server standalone.
 ```
 
-Note: Installing globally will not work with Protractor if you are trying to
+Notes:
+* Installing globally will not work with Protractor if you are trying to
 start a Selenium Standalone server with a "local" or "directConnect", because
 Protractor installs its own version of webdriver-manager as a dependency.
+* Right now there is an incoherence between ChromeDriver and Chrome versions.
```

Comment:
Edit:

Right now there is an issue with downloading the latest ChromeDriver. It might not be compatible with the current version of Chrome.

Reply:
edited